### PR TITLE
Fail a test if the invalid currency isn't reported

### PR DESCRIPTION
**Pull request by choroba**

### CODE REVIEW

**Comment by choroba** [comment]
Or was it skipped intentionally?

File: t/live.t [hunk 1]
```diff
@@ -279,6 +279,8 @@ Charges: {
             is $e->type, 'invalid_request_error', 'error type';
             like $e->message, '/^Invalid currency: zzz/', 'error message';
             is $e->param, 'currency', 'error param';
+        } else {
+            ok 0, 'report invalid currency';
```
Comment (andrewsolomon) [inline t/live.t]: I propose:
fail('report invalid currency');

**Comment by choroba** [comment]
☑️

**Comment by andrewsolomon** [comment]
Thanks @choroba!

**Pull request merged**